### PR TITLE
Creating Element Finance skin file

### PR DESCRIPTION
**Pull request by tinahaibodi**
#### Hi! What is your PR about?

- [x] Add or edit a skin
- [ ] Add custom domain for your space
- [ ] Add alias for migrate your space
- ~Add images for your space~ (deprecated) Add the image URL on the space settings directly
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: skins/element-finance.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  #topnav a, #topnav .text-gray {
+    color: #F5F8FA !important;
+  }
+
+  #topnav {
+    border-bottom: 4px solid #005EBE !important;
+    border-image-slice: 1;
+    border-image-source: linear-gradient(to left, #005EBE, #78D3E2);
+  }
```
Comment (gokaiorg) [inline skins/element-finance.scss:24]: Please keep extra styles only for #topnav.
```suggestion
  #topnav {
    background-color: #005EBE;
    border-bottom: 4px solid #005EBE !important;
    border-image-slice: 1;
    border-image-source: linear-gradient(to left, #005EBE, #78D3E2);
  }
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by tinahaibodi** [comment]
Making changes

File: skins/element-finance.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  --header-bg: #F5F8FA;
+  --block-bg: #F5F8FA;
+
+  #topnav {
+    background-color: #005EBE;
+    border-bottom: 4px solid #005EBE !important;
+    border-image-slice: 1;
+    border-image-source: linear-gradient(to left, #005EBE, #78D3E2);
+  }
+}
```
Comment (gokaiorg) [inline skins/element-finance.scss:17]: The colours used in your skin make the submit button unreadable, please change with others.
Capture d’écran 2021-10-05 à 10 26 44

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**